call.go

```diff
@@ -118,6 +122,13 @@ func sendRequest(ctx context.Context, dopts dialOptions, compressor Compressor,
 	if err != nil {
 		return Errorf(codes.Internal, "grpc: %v", err)
 	}
+	if c.maxSendMessageSize == nil {
+		// TODO(lyuxuan): codes.Internal the right error to return here?
+		return Errorf(codes.Internal, "callInfo maxSendMessageSize field uninitialized(nil)")
+	}
+	if len(outBuf) > *c.maxSendMessageSize {
+		return Errorf(codes.ResourceExhausted, "Sent message larger than max (%d vs. %d)", len(outBuf), *c.maxSendMessageSize)
```

Comment:
C implementation use this sentence. Do we want to be consistent with them or not?
https://github.com/grpc/grpc/blob/master/src/core/ext/filters/message_size/message_size_filter.c#L141